### DISCUSSION TITLE
Inlays length limitation

### DISCUSSION THREAD

**Discussion by qarlosalberto** [microsoft/vscode-discussions#1062] [Extension Development QnA] [answered]
Is there any limitation in inlays length? I'm adding some inlays in Python files (at the end of the line) and VScode shows it as:

```myinli...```

it's cut with "..."

**Comment by theReynald** [comment, marked as answer]
Hi @qarlosalberto.

In VS Code, there isn't a strict limitation on the length of inlays, but the display of inlays can be affected by the available space in the editor window. When the available space is limited, VS Code may truncate the content of the inlay and append "...".

To ensure that your inlays are displayed properly, you might consider the following:

Conciseness: Try to keep the content of your inlays concise. If the content is too long, it may get truncated, especially when the editor window is narrow.

Customization: Check if there are any settings or configurations in your extension that allow you to customize the behavior of inlays, such as maximum length or truncation options.

Dynamic Display: Consider implementing dynamic behavior for your inlays, such as displaying additional information in a tooltip when the user hovers over the truncated content.

User Interface Consideration: Keep in mind the user interface constraints, such as the width of the editor window and the font size, which can affect how much content can be displayed without truncation.

Hopefully that helps. Let me know if you still have questions.

/r

**Reply by qarlosalberto** [reply]
Thanks for the reply! I have this problem:

![imagen](https://github.com/microsoft/vscode-discussions/assets/5955143/e19dfebe-b341-4123-be25-32aa9fb1cd55)

Could I show the inlays without limitation? Is there any setting that I can modify?

Thanks :)
